# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gtkhtml10/compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gtkhtml-1.1.10-orig/src/htmlcluealigned.h gtkhtml-1.1.10/src/htmlcluealigned.h
--- gtkhtml-1.1.10-orig/src/htmlcluealigned.h	2001-05-21 12:47:11.000000000 +0000
+++ gtkhtml-1.1.10/src/htmlcluealigned.h	2006-01-18 12:12:29.872986750 +0000
@@ -37,8 +37,8 @@
 	HTMLClueClass clue_class;
 };
 
-
-extern HTMLClueAlignedClass html_cluealigned_class;
+//
+//extern HTMLClueAlignedClass html_cluealigned_class;
 
 
 void        html_cluealigned_type_init   (void);
